.flake8

```diff
@@ -9,5 +9,6 @@ exclude =
     python/foxglove-schemas-flatbuffer/foxglove_schemas_flatbuffer/*.py,
     python/foxglove-schemas-protobuf/foxglove_schemas_protobuf/*.py,
     python/foxglove-sdk/python/foxglove/_protobuf/*,
+    python/foxglove-sdk/python/foxglove/_foxglove_py/schemas.pyi,
```

Comment:
can we move this now? we format this file right?

Reply:
I'm currently running it through `black` and not flake8, but this is not required; I removed it.